packages/block-serialization-default-parser/parser.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+function bsdp_parse($document ) {
```

Comment:
see related [comment response](
https://github.com/WordPress/gutenberg/pull/8083#discussion_r21283666)9 below.

I'm having some trouble understanding what you wrote @pento. I _hope_ we create a filter to select the parsing function but won't that depend somewhat on having unique names for each possible parse functions?

Reply:
experimented in 064efa58d but I haven't tested it yet

for what it's worth I'd be more comfortable getting this parser in first before making the parser system pluggable just because of the scope of the changes